From cfa16a66c8e31cf21fe53ef0cdffc52ed5dcc433 Mon Sep 17 00:00:00 2001
From: Ricardo Salveti <ricardo@foundries.io>
Date: Fri, 8 Feb 2019 23:12:44 -0200
Subject: [PATCH] utils.c: disable tilde as it is not supported by musl

Config will not be able to search for ~/config, but that is not
recommended anyway.

Signed-off-by: Ricardo Salveti <ricardo@foundries.io>
---
 src/libaktualizr/utilities/utils.cc | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/libaktualizr/utilities/utils.cc b/src/libaktualizr/utilities/utils.cc
index fbd0e485..7fa5f020 100644
--- a/src/libaktualizr/utilities/utils.cc
+++ b/src/libaktualizr/utilities/utils.cc
@@ -31,6 +31,11 @@
 
 #include "logging/logging.h"
 
+/* disable glob tilde as it is not supported by musl */
+#ifndef GLOB_TILDE
+#define GLOB_TILDE 0
+#endif
+
 const char *adverbs[] = {
     "adorable", "acidic",     "ample",        "aromatic",   "artistic", "attractive", "basic",    "beautiful",
     "best",     "blissful",   "bubbly",       "celebrated", "cheap",    "chilly",     "cloudy",   "colorful",
-- 
2.17.1

